sporkfish/searcher/move_ordering/history_heuristic.py

```diff
@@ -0,0 +1,28 @@
+from typing import Dict
+
+import chess
+
+from ...board.board import Board
+from .move_order_heuristic import MoveOrderHeuristic
+
+
+class HistoryHeuristic(MoveOrderHeuristic):
+    def __init__(self, board: Board, history_table: Dict[chess.Move, int]) -> None:
+        MoveOrderHeuristic.__init__(self)
```

Comment:
Sorry this is me being naive - what is the difference between:
```
MoveOrderHeuristic.__init__(self)
```
VS
```
super.__init__(self)
```
in this case? I am used to calling the parent constructor with the second way. 

Had a quick search on google - see [here](https://stackoverflow.com/questions/576169/understanding-python-super-with-init-methods).

Reply:
oh yeah! Sorry didn't notice this... gpt wrote it.

wait! I asked gpt to follow the style of killer move, and killer move also had it. I guess I will change the code in killer too!